### ISSUE TITLE
unable to resolve dependency tree

### ISSUE THREAD

**Issue by cmmishra9** [inferusvv/react-odometerjs#28]
This is not supported by the new React Version 17.0.2
```bash
Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"17.0.2" from the root project
```
```bash
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.3.0" from react-odometerjs@2.1.1
npm ERR! node_modules/react-odometerjs
npm ERR!   react-odometerjs@"*" from the root project
```

**Comment by bezenson** [comment]
Hmm, I think we should change `"react": "^16.3.0"` to `"react": ">= 16.3.0"`

I'll try to do it soon, but you can also create Merge Request with that

**Comment by bezenson** [comment]
Fixed in v2.1.2
